connectors/citrus-openapi/src/main/java/org/citrusframework/openapi/OpenApiTestDataGenerator.java

```diff
@@ -119,7 +120,7 @@ public static String createRandomValueExpression(OasSchema schema, Map<String, O
             }
 
             if (schema.format != null && schema.format.equals("date")) {
-                payload.append("citrus:currentDate()");
+                payload.append("citrus:currentDate('yyyy-MM-dd')");
```

Comment:
for reference:

> date – full-date notation as defined by [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6), for example, 2017-07-21

[source](https://swagger.io/docs/specification/data-models/data-types/#string).